src/Engine/Physics/RigidBody.cpp

```diff
@@ -36,14 +33,18 @@ namespace Physics {
     glm::vec3 RigidBody::Position() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'Position' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)